openshift/settings.sh

```diff
@@ -1,10 +1,9 @@
 export PROJECT_NAMESPACE="7cba16"
 export GIT_URI="https://github.com/bcgov/von-bc-registries-agent.git"
 export GIT_REF="master"
+export SKIP_PIPELINE_PROCESSING=1
 
 # The templates that should not have their GIT referances(uri and ref) over-ridden
 # Templates NOT in this list will have they GIT referances over-ridden
 # with the values of GIT_URI and GIT_REF
-export -a skip_git_overrides="bc-reg-fdw-build.json backup-build.json"
-
-export ignore_templates="backup-deploy"
```

Comment:
This was removed, any specific reason to not ignore this template still?

Reply:
looks like this was also removed by @WadeBarnes according to my git lens. It must be showing up in my PR because of that weirdness we had to do to get around the DCO check